### PR TITLE
code cleanup and font addition

### PR DESCRIPTION
**Pull request by emman4github**
1) Code is cleanup.(onboarding.screen.js) file is created under /src folder.
2) created theme folder which includes:
 * styles.js
 * fonts.js
 * index.js
 * spacing.js
 * sizes.js
 3)In packages.json have installed some packages:
  ->"@expo-google-fonts/lato": "^0.2.2",
  -> "styled-components": "^5.3.6"
  -> "expo-font": "~11.0.1",

### CODE REVIEW

File: nlfyapp/src/features/onboarding.screen.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import Swiper from "react-native-swiper";
+import styled from "styled-components/native";
+
+
+const SafeArea = styled(SafeAreaView)`
+  flex: 1;
+  margin-top: ${StatusBar.currentHeight}px;
+`;
+
+// const wrapper = styled(View)``;
```
Comment (designgeeks777) [inline nlfyapp/src/features/onboarding.screen.js]: Remove commented codes

File: nlfyapp/src/features/onboarding.screen.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+const TextScreen2Orange = styled(Text)`
+  top: 20px;
+  padding-top: 16px;
+  color: ${(props) => props.theme.colors.text.secondary};
+  font-size: ${(props)=> props.theme.fontSizes.caption};
+  font-weight: ${(props)=> props.theme.fontWeights.medium};
+  font-family: ${(props)=> props.theme.fonts.body}; 
+`;
+
+const Button = styled(Pressable)`
```
Comment (designgeeks777) [inline nlfyapp/src/features/onboarding.screen.js:79]: Try to create a reusable style for button ( with pressable ) component

**Review by designgeeks777 — COMMENTED** [review]
Looks ok.Added 2 minor comments

File: nlfyapp/src/features/onboarding.screen.js (hunk 1 above)
Comment (emman4github) [reply nlfyapp/src/features/onboarding.screen.js]: Removed

File: nlfyapp/src/features/onboarding.screen.js (hunk 2 above)
Comment (emman4github) [reply nlfyapp/src/features/onboarding.screen.js:79]: This will be taken care in the next PR

**Review by designgeeks777 — APPROVED** [review]
Reviewed

**Pull request merged**